native/src/constants/layers.ts

```diff
@@ -0,0 +1,54 @@
+import { CircleLayerProps, SymbolLayerProps } from '@react-native-mapbox-gl/maps'
+
+import {
+  circleRadiusLarge,
+  circleRadiusSmall,
+  fontSizeLarge,
+  fontSizeSmall,
+  groupCount,
+  mapMarker,
+  PoiFeature,
+  textOffsetY,
+} from 'api-client'
+import { ThemeType } from 'build-configs/ThemeType'
+
+export const clusterLayer = (theme: ThemeType): CircleLayerProps => ({
```

Comment:
Styling differs between web and native, right? No way to make this more uniform/reuse stuff?

Reply:
Well the libs using different interfaces for the layers. But the result should be quite same. I first thought that we can unify the layers but thats not possible unfortunately